tubular/nominal.py

```diff
@@ -783,12 +789,14 @@ def transform(self, X):
             for c in self.columns:
                 # finding rows with values not in the keys of mappings dictionary
                 unseen_indices[c] = X[~X[c].isin(self.mappings[c].keys())].index
-            X = BaseMappingTransformMixin.transform(self, X)
+            X = super().transform(X)
```

Comment:
Not one for now, but maybe one for the backlog? this section feels a bit clunky and hard to follow. Would be nice not to have the super.transform and self.map_imputation_values twice, and feels confusing that in once case we call the self.check_mappable_rows class method, and in the other we ignore this and call a slightly different but similarly named method.

Feel free to completely ignore this but I have stared at it for too long not to write something

Reply:
I've just noticed your comment that mean response transformer could do with a refactor, I agree

Reply:
I have added a ticket here #149